docs/docs/07-forms.ko-KR.md

```diff
@@ -28,6 +28,9 @@ HTML에서는 `<textarea>` 태그의 값을 설정할 때 `<textarea>` 태그의
 
 모든 DOM 이벤트처럼 `onChange` prop은 모든 네이티브 컴포넌트에서 지원되며 일어난(bubbled) 변경 이벤트를 감시하는데 사용할 수 있습니다.
 
+> 주의:
+>
+> `<input>`, `<textarea>`에는 `onChange`를 사용할 필요가 없고 대신 일반적으로 사용하는 [`oninput`](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/oninput) 이벤트 핸들러를 사용하셔야 합니다.
```

Comment:
하이픈 사이의 문장을 끊어서 번역해야하지 않을까요? 

> For `<input>` and `<textarea>`, `onChange` supersedes the DOM's built-in [`oninput`](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/oninput) event handler.
> — and should generally be used instead of —